test/config.js

```diff
@@ -0,0 +1,26 @@
+const https = require('https')
+
+
+const tcp = {
+  protocol: 'tcp', port: '50001', host: 'electrumx-server.tbtc.svc.cluster.local',
+}
+
+const ssl = {
+  protocol: 'ssl', port: '50002', host: 'testnet1.bauerj.eu',
+}
+
+
+const ws = {
+  protocol: 'ws', port: '50003', host: 'electrumx-server.tbtc.svc.cluster.local',
+}
+
+const wss = {
+  protocol: 'wss', port: '50004', host: 'electrumx-server.tbtc.svc.cluster.local', options: {agent: new https.Agent({rejectUnauthorized: false})},
+}
+
+module.exports = [
```

Comment:
If we're exporting an array/list, I think it'd be nice to name this "configs". Thoughts?

Reply:
I modified it a little bit, please take a look.